content/blog/tidymodels-2021-q1/index.Rmd

```diff
@@ -0,0 +1,135 @@
+---
+output: hugodown::hugo_document
+
+slug: tidymodels-2021-q1
+title: Catch up with tidymodels
+date: 2021-03-10
+author: Julia Silge
+description: >
+    Releases of tidymodels packages in Q1 of 2021 offer new functions for 
+    easier model building and resampling, along with a new package for 
+    resampling spatial data.
+
+photo:
+  url: https://unsplash.com/photos/4Zk45jNyQS4
+  author: Timo Wielink
+
+# one of: "deep-dive", "learn", "package", "programming", or "other"
+categories: [roundup] 
+tags: [tidymodels, parsnip, rsample]
+---
+
+```{r include=FALSE}
+library(ggplot2)
+theme_set(theme_minimal())
+```
+
+
+The [tidymodels](https://www.tidymodels.org/) framework is a collection of R packages for modeling and machine learning using tidyverse principles. There have quite a number of updates and new developments in the tidymodels ecosystem since our [last blog post in December](https://www.tidyverse.org/blog/2020/12/finetune-0-0-1/)! Since that post, tidymodels maintainers have published eight CRAN releases of existing packages. You can install these updates from CRAN with:
```

Comment:
```suggestion
The [tidymodels](https://www.tidymodels.org/) framework is a collection of R packages for modeling and machine learning using tidyverse principles. There have been quite a number of updates and new developments in the tidymodels ecosystem since our [last blog post in December](https://www.tidyverse.org/blog/2020/12/finetune-0-0-1/)! Since that post, tidymodels maintainers have published eight CRAN releases of existing packages. You can install these updates from CRAN with:
```